cli/pkg/workspace/plugin/kubeconfig_test.go

```diff
@@ -407,7 +407,7 @@ func TestUse(t *testing.T) {
 				AuthInfos: map[string]*clientcmdapi.AuthInfo{"test": {Token: "test"}},
 			},
 			destination: "root:foo:bar",
-			wantStdout:  []string{"Current workspace is \"root:foo:bar\""},
+			wantStdout:  []string{"Current workspace is ':root:foo:bar'"},
```

Comment:
am not convinced we want to print out the `:` at all. Let's keep the output always `foo:bar:abc`. The `:` prefix is only for navigation.

Reply:
was one the line with this, so this pushes swings to same :) will remove